### DISCUSSION TITLE
The installation process installs apps by default (firefox, vscode, etc.)?

### DISCUSSION THREAD

**Discussion by VicenteGM123** [end-4/dots-hyprland#573] [Q&A]
Hi, first of all, I'm a complete newbie to linux/arch/lhyprland XD. I did the automatic installation and it's working perfectly! I just have one question, should default applications have been installed? Because I don't have firefox, vscode and the file manager I have doesn't look like the one in the photos of Illogical impulse (I have Dolphin). I wonder if something went wrong during installation or if this installation is only for hyprland's UI configuration. Thank you (the environment is sick)!

**Comment by end-4** [comment]
Hi,
My dotfiles provide mostly the customization of bars and hyprland-related stuff.
Software choice is yours, and I don't want to force it on anyone. So install them if you wish.

**Comment by clsty** [comment]
Welcome to Arch Linux (may I say it). You may find more help at other places, for example Arch Wiki:
- https://wiki.archlinux.org/title/General_recommendations
- https://wiki.archlinux.org/title/List_of_applications

And since you're new to hyprland, also take a look at
- https://wiki.hyprland.org/
